modules/auxiliary/scanner/http/iis_shortname_scanner.rb

```diff
@@ -0,0 +1,252 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Rex::Proto::Http
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'        => 'Microsoft IIS shortname vulnerability scanner',
+        'Description' => %q{
+         The vulnerability is caused by a tilde character "~" in a GET or OPTIONS request, which
+         could allow remote attackers to diclose 8.3 filenames (short names). In 2010, Soroush Dalili
+         and Ali Abbasnejad discovered the original bug (GET request). This was publicly disclosed in
+         2012. In 2014, Soroush Dalili discovered that newer IIS installations are vulnerable with OPTIONS.
+        },
+        'Author'         =>
+          [
+          'Soroush Dalili', # Vulnerability discovery
+          'Ali Abbasnejad', # Vulnerability discovery
+          'MinatoTW <shaks19jais[at]gmail.com>', # Metasploit module
+          'egre55 <ianaustin[at]protonmail.com>' # Metasploit module
+          ],
+        'License'     => MSF_LICENSE,
+        'References'     =>
+          [
+            [ 'URL', 'https://soroush.secproject.com/blog/tag/iis-tilde-vulnerability/' ]
+          ],
+        'Targets' => [[ 'Automatic', {} ]]
+      )
+    )
+
+  register_options([
+          Opt::RPORT(80),
+          OptString.new('PATH', [ true, "The base path to start scanning from", "/" ])
+
+  ])
+  @dirs = []
+  @files = []
+  @threads = []
+  @queue = Queue.new
+  @queue_ext = Queue.new
+  @alpha = 'abcdefghijklmnopqrstuvwxyz0123456789!#$%&\'()-@^_`{}'
+  @found = Array.new
```

Comment:
Can these `@found`, `@found2`, `@found3` variable be given better names? What do they represent? Also, arrays can be defined thusly:

```ruby
  @found = []
  @found2 = []
  @found3 = []
```